Tools/BasicStation-Certificates-Generation/helper-functions

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Helper functions "ecdsaKey", "rootCA", "serverCert" and "clientcert" were inspired by following example
+# in BasicsStation official GitHub repository (https://github.com/lorabasics/basicstation/blob/master/examples/cups/prep.sh)
```

Comment:
nit: consider using a more stable link that's pegged to a tag since `master` can always change and move:

```suggestion
# in BasicsStation official GitHub repository (https://github.com/lorabasics/basicstation/blob/v2.0.5/deps/lgw/prep.sh)
```

---
PS Assuming you took inspiration from the script in v2.0.5.

Reply:
Done, thanks!